src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -271,6 +285,35 @@ export class TerminalService implements ITerminalService {
 		this._extHostsReady[remoteAuthority]!.resolve();
 	}
 
+	public getAvailableProfiles(): ITerminalProfile[] {
+		this._updateAvailableProfiles();
```

Comment:
I still don't think we call this when the primary ext host is ready

Reply:
like this? https://github.com/microsoft/vscode/blob/aef25faa4dd14d10c4a1cefa02136a5994e17750/src/vs/workbench/contrib/terminal/browser/terminalService.ts#L349-L352
not sure what you mean by the primary ext host